src/TodoApp.jsx

```diff
@@ -0,0 +1,33 @@
+import  { useState } from "react";
+import React from "react";
+import AddTodo from "./AddTodo";
+import TodoList from "./TodoList";
+
+function TodoApp() {
+    const [todos, setTodos] = useState([
+        { id: 1, title: "abc",completed: false },
+        { id: 2, title: "xyz",completed: false},
+    ]);
+
+    const [inputValue, setInputValue] = useState(""); 
+    const [editingId, setEditingId] = useState(null);
```

Comment:
you do not need 2 variable to handle this, Use  1 state to handle this, like an object @kaushaki-kumari 
More info you can ask Piyush or Amrit